arangopipe/Dockerfile_TFFE

```diff
@@ -14,6 +14,7 @@ MAINTAINER Joerg Schad <info@arangodb.com>
 ENV GIT_PYTHON_REFRESH=quiet
 RUN apt-get update
 RUN apt-get install -y python-pip
+RUN apt-get install -y python-pip curl
```

Comment:
Could we combine the install commands into one RUN comands? at best together with apt-get update.
This reduces the number of layers for the docker image.